pkg/web/templates/partials/transaction/transaction_list.html

```diff
@@ -19,7 +19,7 @@
       {{ if .Transactions }}
       {{ range .Transactions }}
       <tr class="text-center">
-        <td class="cursor-pointer" onclick="transaction_modal.showModal()" hx-get="/v1/transactions/{{ .ID }}" hx-target="#transaction_modal">
```

Comment:
The click action on the table was replaced by displaying `Preview` in the actions menu.